terraform/layer2-k8s/templates/aws-node-termination-handler-values.yaml

```diff
@@ -1,4 +1,4 @@
 enableSpotInterruptionDraining: true
 enableRebalanceMonitoring: true
 nodeSelector:
-  node.kubernetes.io/lifecycle: spot
+  eks.amazonaws.com/capacityType: SPOT
```

Comment:
Do we need to use it if we use AWS managed node groups?

Reply:
AWS adds `eks.amazonaws.com/capacityType` label for nodes, we need them in order to select SPOT or ONDEMAND nodes for pods

Reply:
I meant, if I don't want to use self-managed node-group, then I don't need aws-node-termination-handler.

Reply:
We still need it for worker groups, I planned to play with it in next PR)